pkg/bucketeer/event/processor_test.go

```diff
@@ -331,6 +331,12 @@ func TestClose(t *testing.T) {
 	}
 }
 
+func TestNewEvent(t *testing.T) {
+	t.Parallel()
+	e := newEvent("", []byte{}, api.EvaluationEventType)
```

Comment:
Please also check the string, bytes, and event type to ensure that arguments are configured correctly.